docker/stolon-development/supervisord.conf

```diff
@@ -1,6 +1,16 @@
 [supervisord]
 user=root
 
+[unix_http_server]
```

Comment:
Instead of this, might we just use pkill?

`docker-compose exec keeper0 pkill -STOP stolon-keeper`
`docker-compose exec keeper0 pkill -CONT stolon-keeper`

Would avoid the whole supervisor shebang? pkill will exit with error if it can't find the process.

Reply:
Done!